doc/user/nhrpd.rst

```diff
@@ -106,11 +156,12 @@ with:
 To start sending these traffic notices out from hubs, use the nhrp
 per-interface directive:
 
-.. code-block:: frr
-
-   interface gre1
-    ip nhrp redirect
+.. index::  ip nhrp redirect
+.. clicmd:: ip nhrp redirect
 
+This enable redierct replies on the NHS similiar to ICMP redirects excpet this is
+managed by the nhrp protocol. This settings allows spokes to communicate with each
+others directly.
 
```

Comment:
to make it more consistent, you should also mention the nflog-group command.
that will make it work spoke to spoke.